app/plugin.go

```diff
@@ -402,6 +411,52 @@ func (a *App) GetPlugins() (*model.PluginsResponse, *model.AppError) {
 	return resp, nil
 }
 
+// GetPluginPublicKeys returns all public keys listed in the config.
+func (a *App) GetPluginPublicKeys() ([]string, *model.AppError) {
+	return a.Config().PluginSettings.SignaturePublicKeyFiles, nil
+}
+
+// GetPublicKey will return the actual public key saved in the `filename` file.
+func (a *App) GetPublicKey(filename string) ([]byte, *model.AppError) {
+	filename += pluginSignaturePublicKeyFileExtention
```

Comment:
Why was the check for extension removed? It seems that it is being passed with the file extension - at least [here](https://github.com/mattermost/mattermost-server/blob/e62959e99de88801ce78ca6bbd8d4a8522d21cc3/app/plugin_test.go#L414)